ext_tables.sql

```diff
@@ -414,3 +414,10 @@ CREATE TABLE tx_caretaker_strategies (
 CREATE TABLE fe_users (
     tx_caretaker_api_key text NOT NULL
 );
+
+#
+# Table structure for table 'tt_address'
+#
+CREATE TABLE tt_address (
+	tx_caretaker_xmpp varchar(30) NOT NULL default ''
```

Comment:
To be in line with core configuration it would be better to use:

tx_caretaker_xmpp varchar(30) DEFAULT '' NOT NULL